src/woeusb

```diff
@@ -2009,14 +2011,14 @@ util_check_function_parameters_quantity(){
 	return 0
 }; declare -fr util_check_function_parameters_quantity
 
-##Check every file in source directory for files bigger than max fat32 file size (~4GB)
+## Check every file in source directory for files bigger than max fat32 file size (~4GB)
 check_for_big_files(){
-	local source_fs_mountpoint="${1}"
+	local -r source_fs_mountpoint="${1}"
 
-	for i in $(find ${source_fs_mountpoint}/ -type f); do
-		if (( $(stat -c "%s" $i) > 4294967295 )); then
-			echo_with_color red "Error: File $i is larger than that supported by the fat32 filesystem. Use NTFS (--target-filesystem NTFS)."
-			exit 1
+	for file in $(find "${source_fs_mountpoint}" -type f); do
+		if (( $(stat -c "%s" $file) > 4294967295 )); then # Max fat32 file size is 2^32 - 1 bytes
```

Comment:
The process substitution will error when the path of `file` contains spaces, quote it with double-quotes.